blob/azureblob/azureblob.go

```diff
@@ -636,7 +640,10 @@ func (b *bucket) ListPaged(ctx context.Context, opts *driver.ListOptions) (*driv
 // SignedURL implements driver.SignedURL.
 func (b *bucket) SignedURL(ctx context.Context, key string, opts *driver.SignedURLOptions) (string, error) {
 	if b.opts.Credential == nil {
-		return "", errors.New("to use SignedURL, you must call OpenBucket with a non-nil Options.Credential")
+		return "", errors.New("azureblob: to use SignedURL, you must call OpenBucket with a non-nil Options.Credential")
+	}
+	if opts != nil && (opts.ContentType != "" || opts.EnforceAbsentContentType) {
```

Comment:
Nit: you don't need to check for `opts == nil` here, the concrete type always passes a non-nil `driver.SignedURLOptions`.

Reply:
Done.